cmd/list_bundles_test.go

```diff
@@ -1,15 +1,69 @@
 package cmd
 
 import (
+	"os"
+	"path/filepath"
+	"strings"
+
 	. "github.com/onsi/ginkgo/v2/dsl/core"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/afero"
 )
 
 var _ = Describe("List Bundles Cmd", func() {
 	When("Calling opcap list bundles", func() {
+		var tmpDir string
+		var gitUrl string
+		var err error
+
+		// create fake test data before each test case
+		BeforeEach(func() {
+			AppFs := afero.NewMemMapFs()
+			tmpDir, err = createFakeRepo(AppFs)
+			Expect(err).ToNot(HaveOccurred())
+			DeferCleanup(os.RemoveAll, tmpDir)
+		})
+		// test list bundles '--from-repo' flag
+		It("should succeed", func() {
+			gitUrl = "https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators.git"
+			_, err = executeCommand(listBundlesCmd(), []string{"--from-repo=" + gitUrl}...)
+			Expect(err).ToNot(HaveOccurred())
+		})
+		// test list bundles cmd '--from-dir' flag
 		It("should succeed", func() {
-			_, err := executeCommand(listBundlesCmd(), []string{}...)
+			str := filepath.Join(tmpDir, "certified-operators")
+			_, err := executeCommand(listBundlesCmd(), []string{"--from-dir=", str}...)
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
 })
+
+// calls create script and creates fake repo containing acc-operator bundles from .test folder for testing
+func createFakeRepo(afs afero.Fs) (string, error) {
+	a := afero.Afero{Fs: afs}
+	tmpDir, err := a.TempDir("", "bundle-test-*")
+	a.MkdirAll("certified-operators/operators", 0o755)
+	copy("../.test/certified-operators/operators/acc-operator", "certified-operators/operators", afs)
```

Comment:
Either explicitly ignore the error with '_ :=' or check the error with Expect(err).ToNot(HaveOccurred()).